sdk/storage/azure-storage-blob/azure/storage/blob/models.py

```diff
@@ -305,54 +305,48 @@ class ContainerPropertiesPaged(Paged):
         begin with the specified prefix.
     :param int results_per_page: The maximum number of container names to retrieve per
         call.
-    :param str marker: An opaque continuation token.
-    """
-    def __init__(self, command, prefix=None, results_per_page=None, marker=None):
-        super(ContainerPropertiesPaged, self).__init__(command, None)
+    :param str continuation_token: An opaque continuation token.
+    """
+    def __init__(self, command, prefix=None, results_per_page=None, continuation_token=None):
+        super(ContainerPropertiesPaged, self).__init__(
+            get_next=self._get_next_cb,
+            extract_data=self._extract_data_cb,
+            continuation_token=continuation_token or ""
+        )
+        self._command = command
         self.service_endpoint = None
         self.prefix = prefix
         self.current_marker = None
```

Comment:
I'm wondering if we should renamed "current_marker" - given that it no longer has a matching "next_marker" - thoughts?

Reply:
Good point, let's make it "marker" for fidelity.

Reply:
fixed